src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -1462,8 +1462,8 @@ unprivileged_init( fd_topo_t *      topo,
     ctx->bmtree[i]           = FD_SCRATCH_ALLOC_APPEND( l, FD_BMTREE_COMMIT_ALIGN, FD_BMTREE_COMMIT_FOOTPRINT(0) );
   }
   void * tpool_worker_mem    = FD_SCRATCH_ALLOC_APPEND( l, FD_SCRATCH_ALIGN_DEFAULT, tile->replay.tpool_thread_count * TPOOL_WORKER_MEM_SZ );
-  ulong  total_spad_size     = fd_ulong_align_up( 128UL * FD_ACC_TOT_SZ_MAX, FD_SPAD_ALIGN );
-  void * spad_mem            = FD_SCRATCH_ALLOC_APPEND( l, fd_spad_align(), tile->replay.tpool_thread_count * total_spad_size );
+  ulong  thread_spad_size    = fd_spad_footprint( 128UL * fd_ulong_align_up( FD_ACC_TOT_SZ_MAX, FD_ACCOUNT_META_ALIGN ) );
```

Comment:
I'm pretty sure we want each borrowed allocation to be aligned with the start of an spad frame. So, we should make the footprint such that each account's allocation is aligned up to FD_SPAD_ALIGN